<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perna de pau</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(174, 217, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>293</width>
      <height>172</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/senac.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>195</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matricula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>230</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>265</y>
      <width>52</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>300</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telefone:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>195</y>
      <width>184</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Ex: 25555</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>184</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Ex: José Carlos Da Silva</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="box3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>265</y>
      <width>184</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Ex: jorginho@gmail.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>184</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string>(00)00000-0000</string>
    </property>
    <property name="text">
     <string>()-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Ex: (41)9 9599-9888</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt1">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>351</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maculino</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>351</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Feminino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="env2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>156</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 255, 61);
alternate-background-color: rgb(185, 255, 197);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Exibir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="env1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>156</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 255, 61);
alternate-background-color: rgb(185, 255, 197);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>351</y>
      <width>136</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Não Informar</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>box1</zorder>
   <zorder>box2</zorder>
   <zorder>box3</zorder>
   <zorder>box4</zorder>
   <zorder>bt1</zorder>
   <zorder>bt2</zorder>
   <zorder></zorder>
   <zorder>env2</zorder>
   <zorder>env1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
